client/stylesheets/_modal.scss

```diff
@@ -1136,7 +1136,45 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 /*~~ Join Button End ~~*/
 
 .modJoinWrapper {
+	width: 70% !important;
+	margin: 60px auto;
+}
+
+.modJoinWrapper h1 {
+	font-size: 2.4rem;
+	font-weight: 400;
+	margin-bottom: 0;
+}
+
+.modJoinWrapper ul li {
```

Comment:
Merge rule `.modJoinWrapper ul li` with rule on line 1138<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `modJoinWrapper` should be written in lowercase with hyphens